src/wallet/rpcwallet.cpp

```diff
@@ -2255,7 +2255,7 @@ static UniValue listlockunspent(const JSONRPCRequest& request)
                     {RPCResult::Type::STR_HEX, "txid", "The transaction id locked"},
                     {RPCResult::Type::NUM, "vout", "The vout value"},
                 }},
-            }
+                    }
```

Comment:
Looks like formatting is broken now, used to be better before... Is it expecting? 
And not only there, but in other places.